lib/mission_control/jobs/engine.rb

```diff
@@ -67,10 +67,9 @@ class Engine < ::Rails::Engine
 
       console do
         require "irb"
-        require "irb/context"
```

Comment:
Newer versions of IRB require `irb/context` by default, so this won't be necessary.